### ISSUE TITLE
[FR] Implement ISR

### ISSUE THREAD

**Issue by pinarruiz** [sladg/nextjs-lambda#81]
I have seen ISR is on the way, is there any ETA on when it will be implemented?

**Comment by sladg** [comment]
Heya! I put it on side without ETA. Ultimately, i would need example project implementing it extensively so I can try it out :)

**Comment by pinarruiz** [comment]
Would you like me to prepare a sample project with ISR so it can be tested?

**Comment by sladg** [comment]
@pinarruiz yeah! if you have some project on hand we could use for testing and debugging, that would be perfect!

**Comment by pinarruiz** [comment]
I have uploaded https://github.com/pinarruiz/nextjs-isr-test-project and invited you as a collaborator, the project will create n amount of random entries and revalidate every 10 seconds.

**Comment by sladg** [comment]
@pinarruiz quick followup. The way to implement ISR nowadays is going with `incrementalCacheHandlerPath`. I'm trying to document these over here: https://github.com/sladg/doc-next-lambda.
